### PR TITLE
Branch david mc bee

### PR DESCRIPTION
**Pull request by davidmcbee**

### CODE REVIEW

**Review by bksuro — CHANGES_REQUESTED** [review]
remove the 'Branch_David_McBee' and 'main' from queries.sql file

File: queries.sql [hunk 1]
```diff
@@ -92,4 +92,154 @@ AS
 SELECT * FROM all_prod;
 SELECT * FROM all_prices;
 SELECT * FROM all_market;
-SELECT * FROM climate;
\ No newline at end of file
+Branch_David_McBee
```
Comment (davidmcbee) [inline queries.sql:95]: will drop line

File: queries.sql [hunk 2]
```diff
@@ -92,4 +92,154 @@ AS
...
+	clim.sp09,
+	clim.sp12,
+	clim.sp24,
+	clim.tmin,
+	clim.tmax
+FROM all_prod AS prod
+INNER JOIN climate AS clim USING (year_month);
+=======
+SELECT * FROM climate;
+main
```
Comment (davidmcbee) [inline queries.sql:245]: dropping

**Review by bksuro — APPROVED** [review]
Looks good. Reviewed.

**Pull request merged**